releasenotes/notes/kubeconfig_delegate-2206c2e1971c7a1c.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    'delegate_to:' localhost was needed on this play to work
```

Comment:
Bear in mind this would appear in release notes without context. We don't publish release notes for our forks, so I think we can just ignore it.